src/core/include/openvino/op/reverse_sequence.hpp

```diff
@@ -38,24 +38,26 @@ class OPENVINO_API ReverseSequence : public Op {
     int64_t get_origin_batch_axis() const {
         return m_batch_axis;
     }
-    void set_batch_axis(int64_t batch_axis) {
-        m_batch_axis = batch_axis;
-    }
+    void set_batch_axis(int64_t batch_axis);
     size_t get_sequence_axis() const {
         return m_normalized_seq_axis;
     }
     int64_t get_origin_sequence_axis() const {
         return m_seq_axis;
     }
-    void set_sequence_axis(int64_t sequence_axis) {
-        m_seq_axis = sequence_axis;
-    }
+    void set_sequence_axis(int64_t sequence_axis);
 
 private:
     int64_t m_batch_axis;
     int64_t m_seq_axis = 1;
     size_t m_normalized_batch_axis;
     size_t m_normalized_seq_axis;
+    template <class T>
+    friend void shape_infer(const ReverseSequence* op,
+                            const std::vector<T>& input_shapes,
+                            std::vector<T>& output_shapes);
+
+    void normal_reverse_sequence_axis(ReverseSequence* op);
```

Comment:
Can we rename it to the `normalize_axes`?

Reply:
yes, of course, will change it